### PR TITLE
EG-531 Send Confirmation processing

### PR DESCRIPTION
**Pull request by LesWilson**
Code for the send confirmation screen

### CODE REVIEW

File: app/models/ApprovalStatus.scala [hunk 1]
```diff
@@ -30,4 +35,28 @@ object ApprovalStatus extends Enumerable.Implicits {
   val values: Seq[ApprovalStatus] = Seq(SubmittedFor2iReview, SubmittedForFactCheck, WithDesignerForUpdate, ApprovedForPublishing)
 
   implicit val enumerable: Enumerable[ApprovalStatus] = Enumerable(values.map(v => v.toString -> v): _*)
+
+  def get2iReviewResultRadioItems(form: Form[_])(implicit messages: Messages): Seq[RadioItem] = {
```
Comment (cmrhmrc) [inline app/models/ApprovalStatus.scala:39]: these fns should be with view related code, probably the controller

File: app/services/ReviewService.scala [hunk 2]
```diff
@@ -34,4 +32,10 @@ class ReviewService @Inject() (reviewConnector: ReviewConnector) {
 
   }
 
+  def approval2iReviewComplete(id: String, status: ApprovalStatus)(implicit ex: ExecutionContext, hc: HeaderCarrier): Future[RequestOutcome[Unit]] = {
+
+    val changeInfo = ApprovalProcessStatusChange("userId", "userName", status)
```
Comment (cmrhmrc) [inline app/services/ReviewService.scala:37]: what are these strings ?? "userId" and "userName"

File: app/views/twoeye_review_result.scala.html [hunk 3]
```diff
@@ -42,7 +42,7 @@
     @inputRadio(
       form = form,
       legend = "2iReviewResult.heading",
-      items = TwoEyeReviewResultType.options(form),
+      items = ApprovalStatus.get2iReviewResultRadioItems(form),
```
Comment (cmrhmrc) [inline app/views/twoeye_review_result.scala.html:45]: see ApprovalStatus comment

File: app/services/ReviewService.scala (hunk 2 above)
Comment (LesWilson) [reply app/services/ReviewService.scala:37]: we don't have the Action that gets these yet - they are in a different PR

**Pull request merged**